docs/source/index.rst

```diff
@@ -109,7 +109,7 @@ Details about the optimized contraction order can be explored:
 
     >>> from opt_einsum import contract_path
 
-    >>> path_info = oe.contract_path('pi,qj,ijkl,rk,sl->pqrs', C, C, I, C, C)
+    >>> path_info = contract_path('pi,qj,ijkl,rk,sl->pqrs', C, C, I, C, C)
```

Comment:
Thanks for catching this, it looks like we use `import opt_einsum as oe` much more than individual function imports in the docs. We need to do a general homogenization pass, but for this example at least can you change line 110 instead of 112?

Reply:
I did, thank you for the review.